### ISSUE TITLE
Upgrade Trezor Connect to 9.4.0+

### ISSUE THREAD

**Issue by kimlwh** [caravan-bitcoin/caravan#161]
Could Trezor Connect be upgraded to 9.4.7 (latest version as of today) or any version above 9.4.0?

Initializing newer versions of Trezor Connect with `TrezorConnect.init({coreMode: "auto", ...})` now allows Trezor devices to be used without any local bridge software necessary, like the other devices Caravan supports. This is especially useful on platforms and devices where only the web is available or local software isn't, this compatibility has always been Caravan's biggest selling point as a stateless coordinator.

Was there a compatibility reason as to why Trezor Connect is frozen at version 9.1.12? I believe this is a worthwhile upgrade, the only meaningful one in fact since the current version Caravan is currently using.

Thank you.

**Comment by bucko13** [comment]
Glad the problem isn't that trezor connect is now broken!

This does seem like a meaningful upgrade as well! Do you have a link to more docs on this upgrade? I didn't see anything in the 9.4.7 changelog. 

> Was there a compatibility reason as to why Trezor Connect is frozen at version 9.1.12?

No reason for that version in particular. That was just the last one we explicitly tested. The reason that we pinned though is that if we used the default to use the `latest` then trezor could inadvertently brick users since you access connect JIT and whatever version is latest is what you use. This has happened before where something was broken and users couldn't sign. We pinned to an earlier version in a release and reported the bug to trezor which was quickly released, but it felt like a precarious position to maintain so pinning and choosing to upgrade (and testing first) seemed preferable. 

would you be willing to put up a PR that bumps the version? We can confirm everything still works and then upgrade.

**Comment by kimlwh** [comment]
>Glad the problem isn't that trezor connect is now broken!

Yes I may have been confusing it with the development/hosted version that doesn't have cross-origin issues.

There doesn't seem to be much documentation other than the addition of the `coreMode` option which is documented under the [`init`](https://connect.trezor.io/9/methods/other/init/) method TrezorConnect and in the [changelog](https://connect.trezor.io/9/changelog/) under 9.4.0.

> would you be willing to put up a PR that bumps the version? We can confirm everything still works and then upgrade.

I will try, though I have no means of local development at the moment and little JavaScript knowledge. A couple of other changes are likely required such as removing the pinned `connectSrc`.

**Comment by bucko13** [comment]
Could you add a changeset version bump for caravan/wallets and caravan-coordinator. I think minor version for both makes sense as this is a new feature. See the readme [here](https://github.com/caravan-bitcoin/caravan?tab=readme-ov-file#changesets) for more details

**Comment by kimlwh** [comment]
It looks like that requires npm and I don't have easy access to any development environment currently, all these changes have just been made with GitHub's web editor, which I don't think even allows you to squash commits. Is it possible for you to finalize the changes?

**Comment by kimlwh** [comment]
Closed by #163.
